components/list-item.json

```diff
@@ -15,6 +15,7 @@
       "background": "@theme-background-primary-hover",
       "textprimary": "@theme-text-primary-normal",
       "textsecondary": "@theme-text-secondary-normal",
+      "textmissed": "@theme-text-error-normal",
```

Comment:
There was no hover/pressed/disabled states for textmissed, introduced that so that they can be over-written for High Contrast